assets/javascripts/src/modules/giraffe.es6

```diff
@@ -128,60 +131,68 @@ function ophanId(){
     })
 }
 
-function carousel() {
-    const HIDDEN = 'js-hidden-tablet';
-    let $contribution = $('.js-contribution');
-    let $contributionButton = $('.js-advance',$contribution);
-    let $details = $('.js-details');
-    let $detailsButton = $('.js-advance', $details);
-    let $pay = $('.js-payment');
-    let all = Array.from($('.form__column'));
+function skipAmount() {
+    let skip = getSkipAmountFromQueryString( window.location.search );
```

Comment:
This returns the value of skipAmount but it is never used for anything.
Maybe I'm misunderstanding how this is supposed to work but shouldn't this value be used to actually set the selected amount or will this come form somewhere else?
right now if you when to the contributions frontend with a value for skipAmount it will skip the amounts part but it will use the default highlighted amount.